storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -381,7 +391,7 @@
                 (:storm-id assignment)
                 port
                 id
-                mem-onheap)
+                resources)
```

Comment:
Let's remove the binding for `mem-onheap` above since we are no longer using it.